openapi3/schema_formats_test.go

```diff
@@ -75,3 +75,33 @@ func TestFormatCallback_WrapError(t *testing.T) {
 
 	delete(SchemaStringFormats, "foobar")
 }
+
+func TestReversePathInMessageSchemaError(t *testing.T) {
+	DefineIPv4Format()
+
+	SchemaErrorDetailsDisabled = true
```

Comment:
From a quick glance I believe tests fail due to tests being run in parallel and multiple times. This is to detect race conditions. 

Here, there this obvious racey code. So I recommend to move this test to another package (as is done with a few other tests). Something with `_test` IIRC. 

Otherwise LGTM!

Reply:
The tests will run in parallel if we explicitly specify this with `t.Parallel()`: https://pkg.go.dev/testing#T.Parallel